### DISCUSSION TITLE
LunchBox Thanks You!

### DISCUSSION THREAD

**Discussion by BustedTaint** [mathoudebine/turing-smart-screen-python#215] [General]
First @mathoudebine **_Thank You!_** 
Second @everyone that has contributed, **_thank you as well!_**
Like many others, I purchased the 3.5 inch screen assuming it was Aida64 compatible, since it was advertised as such. After finding that it is not Aida64 compatible, I downloaded the software from turing, but seemed like it may have some malware or at least some sus activities. Honestly I had some other issues going when I originally downloaded it, but none the less, I sought out an alternative, and discovered this project/repository. I'm not a programmer, but fumbled through it. In the end only ended up with one NaN error, which I admit I went to Chat GPT with, and resolved it by using math and placing a zero for NaN on the disk activity. I think Libre throws an od NaN every once in a while for my disk activity.
So this screen is for a build I'm taking to DreamHack. I dubbed it LunchBox, as it's about the size of a lunch box. I just wanted a screen on the front to mostly monitor temps, utilization, and power consumption, ....and this is what I ended up with. Thank you again for the awesome project/repository!
![LunchBox](https://user-images.githubusercontent.com/17506729/236110812-132d3215-332b-4f80-ade7-4a5d3b203eeb.jpg)

**Comment by mathoudebine** [comment]
Hi @BustedTaint
Always a pleasure to see what people have done with this project! A lot of people were disappointed by the fact it was not really compatible with Aida64, and I understand the concern of running software that you don't trust on your computer. This project started as an open-source alternative and it grew really fast thanks to the community
The display integrates really well in your build! I like the fact that you added power sensors for CPU & GPU, it might be something we could add to this project in the future
Don't hesitate to share your nice theme with the community if you'd like: https://github.com/mathoudebine/turing-smart-screen-python/discussions/categories/themes
Thanks for sharing, have fun in DreamHack!

**Comment by yanndaok** [comment]
Hi @BustedTaint 

I really love this theme you've done, do you intend to share it ?
Thx in advance

**Comment by DJRoby19** [comment]
Yes, it would be nice if you shared it, thanks
